### DISCUSSION TITLE
message Header Custom View

### DISCUSSION THREAD

**Discussion by rashdan** [MessageKit/MessageKit#1664] [Q&A]
I want to add custom message header view in my chat . can anyone help me  how I can use it .
two labels and one imageView like the image below. 
<img width="355" alt="Screenshot 2022-01-10 at 8 16 29 AM" src="https://user-images.githubusercontent.com/1507500/148729961-1cf038f8-98c6-4deb-ae44-50a53287b41d.png">

**Comment by Kaspik** [comment]
I guess you can implement completely custom cell and it could be part of that?

**Comment by erwanmace** [comment]
You can use messagetoplabel which is an attributed string to show the name and time in different Colors or fonts. 
you can then set the alignment of the avatar with edge insets, as well as of the messagetoplabel to position them the way you want
